package.json

```diff
@@ -105,15 +90,15 @@
     "terra-markdown": "^2.0.0",
     "terra-slide-panel": "^2.0.1",
     "terra-theme-provider": "^2.0.0",
+    "terra-toolkit": "git://github.com/cerner/terra-toolkit.git#webpack4",
+    "webpack-merge": "^4.1.2",
     "xfc": "^1.2.1"
   },
   "peerDependencies": {
     "babel-polyfill": "^6.23.0",
```

Comment:
I think terra-dev-site should provide the babel polyfill version it requires instead of having this be a peer dependency. Thoughts?

Reply:
Actually, never mind, this is provided by toolkit, I can just remove it.

Reply:
Agreed since its included in the default webpack config in terra-dev-site